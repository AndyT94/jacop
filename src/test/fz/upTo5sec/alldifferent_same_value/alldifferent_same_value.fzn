array [1..4] of var int: x:: output_array([1..4]) = [7,3,1,5];
array [1..4] of var int: y:: output_array([1..4]) = [1,3,1,7];
solve  satisfy;
