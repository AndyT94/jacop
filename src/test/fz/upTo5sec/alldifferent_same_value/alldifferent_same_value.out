x = array1d(1..4, [7, 3, 1, 5]);
y = array1d(1..4, [1, 3, 1, 7]);
----------